### PR TITLE
add ocrd_olahd_client

### PR DESCRIPTION
**Pull request by kba**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -79,3 +79,6 @@
 [submodule "ocrd_wrap"]
 	path = ocrd_wrap
 	url = https://github.com/bertsky/ocrd_wrap.git
+[submodule "ocrd_olahd_client"]
+	path = ocrd_olahd_client
+	url = https://github.com/OCR-D/ocrd_olahd_client
```
Comment (stweil) [inline .gitmodules:84]:
```suggestion
	url = https://github.com/OCR-D/ocrd_olahd_client.git
```

**Comment by stweil** [comment]
@kba, we should have a LICENSE file for the new `ocrd_olahd_client`.

**Review by bertsky — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -754,9 +761,9 @@ endif
 docker-%-git: PIP_OPTIONS = -e
 
 # Minimum-size selection: use Ocropy binarization, use Tesseract from PPA
-docker-mini%: DOCKER_MODULES = core ocrd_cis ocrd_fileformat ocrd_im6convert ocrd_pagetopdf ocrd_repair_inconsistencies ocrd_tesserocr ocrd_wrap tesserocr workflow-configuration
+docker-mini%: DOCKER_MODULES = core ocrd_cis ocrd_fileformat ocrd_im6convert ocrd_pagetopdf ocrd_repair_inconsistencies ocrd_tesserocr ocrd_wrap tesserocr workflow-configuration ocrd_olahd_client
 # Medium-size selection: add Olena binarization and Calamari, use Tesseract from git, add evaluation
```
Comment (stweil) [inline Makefile:765]: Sorting can be done in a follow-up commit.

**Review by stweil — APPROVED** [review]
> @kba, we should have a LICENSE file for the new ocrd_olahd_client.

I added two commits to fix the submodule urls and update ocrd_olah_client with a license file.

**Pull request merged**